samples/test/support/wait/waitForDisplayed.ts

```diff
@@ -15,9 +15,24 @@ export default async (selector: Selector, falseCase: any = false) => {
     const ms = 10000;
 
     const el = await $(selector);
-    await el.waitForDisplayed({
-        timeout: ms,
-        reverse: Boolean(falseCase),
-    });
+
```

Comment:
what issue this change is trying to resolve?

Reply:
I had problems with trying to figure out why `waitForDisplayed` fails with timeout error in CI (expected element was not found by selector on page after 10s). To debug this I return page HTML to console for investigation.  In this case problem was with credentials and I've found `Authentication failed` in HTML

Reply:
I would suggest to keep the original function unchanged. And for debugging, you probably can start with the sample app in the browser.

Reply:
Reverted.

Reply:
Reverted.